src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -614,6 +614,14 @@ public function text(/* string $default = null, bool $normalizeWhitespace = fals
 
         $text = $this->getNode(0)->nodeValue;
 
+        if (\func_num_args() <= 1) {
+            if ($text !== trim(preg_replace('/(?:\s{2,}+|[^\S ])/', ' ', $text))) {
+                @trigger_error(sprintf('Set $normalizeWhitespace to "false" when calling "%s" to retrieve the non-normalized version of the text.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
`"%s()" will normalize whitespaces by default in Symfony 5.0, set the 2nd "$normalizeWhitespace" argument to false to retrieve the non-normalized version of the text.`